Bio/PDB/PQRParser.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (C) 2002, Thomas Hamelryck (thamelry@binf.ku.dk)
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+
+"""Parser for PQR files."""
+
+from __future__ import print_function
+
+import warnings
+
+try:
+    import numpy
+except ImportError:
+    from Bio import MissingPythonDependencyError
+    raise MissingPythonDependencyError(
+        "Install NumPy if you want to use the PQR parser.")
+
+from Bio.File import as_handle
+from Bio.PDB.PDBExceptions import PDBConstructionException
+from Bio.PDB.PDBExceptions import PDBConstructionWarning
+from Bio.PDB.StructureBuilder import StructureBuilder
+from Bio.PDB.parse_pdb_header import _parse_pdb_header_list
+from Bio.PDB.PDBParser import PDBParser
+
+
+class PQRParser(PDBParser):
+    """Create a PQRParser object."""
+
+    def _parse_coordinates(self, coords_trailer):
+        """Parse the atomic data in the PQR file (PRIVATE)."""
+        local_line_counter = 0
+        structure_builder = self.structure_builder
+        current_model_id = 0
+        # Flag we have an open model
+        model_open = 0
+        current_chain_id = None
+        current_segid = None
+        current_residue_id = None
+        current_resname = None
+        for i in range(0, len(coords_trailer)):
+            line = coords_trailer[i].rstrip('\n')
```

Comment:
We prefer double quotes over single quotes (something the code formatting tool ``black`` does, which we are working towards adopting for the whole of Biopython).